Formula/felinks.rb

```diff
@@ -0,0 +1,127 @@
+
+    # Allow screen-256color to be a VT100 variant, just like xterm-256color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-256color.type", NULL) = TERM_VT100;',
+      'get_opt_int("terminal.xterm-256color.type", NULL) = TERM_VT100;
+       get_opt_int("terminal.screen-256color.type", NULL) = TERM_VT100;'
+
+    # Allow screen-256color to have 256 colors, just like xterm-256color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-256color.colors", NULL) = COLOR_MODE_256;',
+      'get_opt_int("terminal.xterm-256color.colors", NULL) = COLOR_MODE_256;
+       get_opt_int("terminal.screen-256color.colors", NULL) = COLOR_MODE_256;'
+
+    # Allow screen-256color to underline, just like xterm-256color
+    inreplace "src/config/options.c",
+      'get_opt_bool("terminal.xterm-256color.underline", NULL) = 1;',
+      'get_opt_bool("terminal.xterm-256color.underline", NULL) = 1;
+       get_opt_bool("terminal.screen-256color.underline", NULL) = 1;'
+
+    # Allow screen-88color to be a VT100 variant, just like xterm-88color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-88color.type", NULL) = TERM_VT100;',
+      'get_opt_int("terminal.xterm-88color.type", NULL) = TERM_VT100;
+       get_opt_int("terminal.screen-88color.type", NULL) = TERM_VT100;'
+
+    # Allow screen-88color to have 88 colors, just like xterm-88color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-88color.colors", NULL) = COLOR_MODE_88;',
+      'get_opt_int("terminal.xterm-88color.colors", NULL) = COLOR_MODE_88;
+       get_opt_int("terminal.screen-88color.colors", NULL) = COLOR_MODE_88;'
+
+    # Allow screen-88color to underline, just like xterm-88color
+    inreplace "src/config/options.c",
+      'get_opt_bool("terminal.xterm-88color.underline", NULL) = 1;',
+      'get_opt_bool("terminal.xterm-88color.underline", NULL) = 1;
+       get_opt_bool("terminal.screen-88color.underline", NULL) = 1;'
+
+    # Allow screen-color to be a VT100 variant, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-color.type", NULL) = TERM_VT100;',
+      'get_opt_int("terminal.xterm-color.type", NULL) = TERM_VT100;
+       get_opt_int("terminal.screen-color.type", NULL) = TERM_VT100;'
+
+    # Allow screen-color to have colors, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_int("terminal.xterm-color.colors", NULL) = COLOR_MODE_16;',
+      'get_opt_int("terminal.xterm-color.colors", NULL) = COLOR_MODE_16;
+       get_opt_int("terminal.screen-color.colors", NULL) = COLOR_MODE_16;'
+
+    # Allow screen-color to underline, just like xterm-color
+    inreplace "src/config/options.c",
+      'get_opt_bool("terminal.xterm-color.underline", NULL) = 1;',
+      'get_opt_bool("terminal.xterm-color.underline", NULL) = 1;
+       get_opt_bool("terminal.screen-color.underline", NULL) = 1;'
```

Comment:
All this needs to be submitted upstream

Reply:
Will do.

Reply:
I'll drop this here and try to get it taken care of upstream.